src/saving/SaveList.cs

```diff
@@ -410,12 +430,9 @@ private void LoadSave()
         }
 
         SaveHelper.LoadSave(saveToBeLoaded);
+
         EmitSignal(nameof(OnSaveLoaded), saveToBeLoaded);
         saveToBeLoaded = null;
-    }
-
-    private void OnSaveLoadedWithoutProblems(string saveName)
-    {
-        EmitSignal(nameof(OnSaveLoaded), saveName);
+        isLoadingSave = false;
```

Comment:
I was worried that this gets unset for one frame when the loading process hasn't taken hold yet, but maybe that doesn't happen as long as the performed invoke runs on the same frame as this input then the loading screen will get displayed immediately over this save list.